packages/web/modals/ibc-transfer.tsx

```diff
@@ -32,6 +38,21 @@ export const IbcTransferModal: FunctionComponent<ModalBaseProps & IbcTransfer> =
 
     const [didAckWithdrawRisk, setDidAckWithdrawRisk] = useState(false);
 
+    const transferHealth = isWithdraw
+      ? queriesExternalStore.queryIbcStatuses.getIbcStatus(
+          "withdraw",
+          counterpartyChainId,
+          props.sourceChannelId
+        )
+      : queriesExternalStore.queryIbcStatuses.getIbcStatus(
+          "deposit",
+          counterpartyChainId,
+          props.sourceChannelId
+        );
+    const transferPathHealth =
+      (transferHealth == IbcStatus.Congested && "Congested") ||
```

Comment:
Please use `===` instead of `==` in js. It checks the types as well.

https://www.scaler.com/topics/javascript/difference-between-double-equals-and-triple-equals-in-javascript/